testsuite/forge/src/main.rs

```diff
@@ -184,7 +185,7 @@ impl Test for EmitTransaction {
 impl NetworkTest for EmitTransaction {
     fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
         let duration = Duration::from_secs(10);
-        let rng = ::rand::rngs::StdRng::from_seed([0; 32]);
+        let rng = ::rand::rngs::StdRng::from_seed(OsRng.gen());
```

Comment:
Don't use the OsRng, instead you should use the Rng that is inside the `ctx` struct